users/static/users/center_style.css

```diff
@@ -1,13 +1,101 @@
-table {
-  margin-left:auto;
-  margin-right:auto;
+body{
+    background-color: #f2f2f2;
+    width:100%;
+    font-family: 'Roboto', sans-serif;
+    margin: 0;
 }
 
-th, td.padding {
-  padding: 15px;
+#menu{
+  margin-left: -15rem;
+  width: 15%;
+  height: 100%;
+  -webkit-transition: margin .25s ease-out;
+  -moz-transition: margin .25s ease-out;
+  -o-transition: margin .25s ease-out;
+  background-color: #2c77bc;
+  position: fixed;
+  top:0;
+  color: white;
+  transition: 0.5s;
+
+  -webkit-box-shadow: 2px 0px 5px 0px rgba(0,0,0,0.77);
+  -moz-box-shadow: 2px 0px 5px 0px rgba(0,0,0,0.77);
+   box-shadow: 2px 0px 5px 0px rgba(0,0,0,0.77);
+ }
+#menu a{
+    text-decoration: none;
+    color: white;
+    font-size: 150%;
+}
+#avatar{
+    text-align: center;
+}
+#user-type{
+    text-align: center;
+    font-size: 150%;
+    visibility: visible;
+}
+
+#logo-menu{
+    position:absolute;
+    bottom:0%;
+    max-height: 13%;
+    padding-bottom: 40%;
+    background-color: #e9efff;
+    width: 100%;
+}
+
+#logo-menu img{
+    width: 55%;
+    vertical-align: 0;
+}
+#logo-menu input{
+    margin-top: 2rem;
+    margin-left: 2.1rem;
+}
+
+#page-content-wrapper{
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
+}
+.href-blocks{
+    margin-top: 35px;
+}
+.img{
+    float:left;
+    margin-left: 3.5rem;
 }
 
-div {
-  text-align: center;
-  margin-bottom: 0;
+.img img{
+    width: 50%;
 }
+
+.text{
+    float:left;
+    color: #a3afd3;
+}
+
+.text:hover{
+    color: white;
+    transition: 0.25s;
+}
+
+#logout{
+    position:fixed;
+    bottom: 16%;
+    width: 15%;
+    overflow: hidden;
+}
+
+@media (min-width: 1550px) {
+  #menu {
+    margin-left: 0;
+  }
+
+
+  #wrapper.toggled #menu {
+    margin-left: -35rem;
+  }
+}
+
```

Comment:
Please add spaces between names and style declarations

Reply:
Please also add everywhere where it is needed space between argument and `:`

Reply:
Done